### PR TITLE
ZSUI-865: Show warnings for known issues when configuring grid.

### PR DESCRIPTION
**Pull request by php1990**

### CODE REVIEW

File: demo.js [hunk 1]
```diff
@@ -193,54 +193,38 @@ var saveEditedJSON = function () {
...
 
+	// Auto placement warning
+	for(var i=0;i<config.cells.length;i++){
+		if(config.cells[i].colSpan>config.cols.length){
+			createAlert("Auto-placement of grid cells is not supported on IE11. Hence, Powergrid restricts spanning of cells beyond configured number of columns. See <a target='_blank' href='https://github.com/ZS/powergrid/issues/9'>details</a>");
+			break;
+		}
+	}
+
+	//TODO:Additional warning scenarios can be added here.
```
Comment (php1990) [inline demo.js]: Further conditions can be added here to show additional warnings.

Comment (MikeVaz) [reply demo.js]: it is a good start. But we need to drive this alerts from json config file. They can't be hard-coded in the demo.js

**Comment by MikeVaz** [comment]
Could you please provide steps how to test it. Cause it not that straight forward.

File: demo.js (hunk 1 above)
Comment (php1990) [reply demo.js]: Nice point. We can have a separate config file for storing warning texts.

**Comment by php1990** [comment]
Steps to test warning:
- Go to "Configure Grid".
- For any of the cells, add a "colSpan" value more than the number of grid columns configured. For example, in our default config, setting "colSpan" as 5 (wiz. more than columns configured - 4) would show up a warning after pressing "OK".

**Pull request merged**